Unqualified Dublin Core RDF.js

```diff
@@ -82,42 +108,44 @@ function doExport() {
 		
 		// publisher/distributor
 		if (item.publisher) {
-			Zotero.RDF.addStatement(resource, dc+"publisher", item.publisher, true);
-		} else if (item.distributor) {
-			Zotero.RDF.addStatement(resource, dc+"publisher", item.distributor, true);
-		} else if (item.institution) {
-			Zotero.RDF.addStatement(resource, dc+"publisher", item.institution, true);
+			Zotero.RDF.addStatement(resource, dc + "publisher", item.publisher, true);
+		}
+		else if (item.distributor) {
+			Zotero.RDF.addStatement(resource, dc + "publisher", item.distributor, true);
+		}
+		else if (item.institution) {
+			Zotero.RDF.addStatement(resource, dc + "publisher", item.institution, true);
 		}
 		
 		// date/year
 		if (item.date) {
-			Zotero.RDF.addStatement(resource, dc+"date", item.date, true);
+			Zotero.RDF.addStatement(resource, dc + "date", item.date, true);
 		}
 		
 		// ISBN/ISSN/DOI
 		if (item.ISBN) {
-			Zotero.RDF.addStatement(resource, dc+"identifier", "ISBN "+item.ISBN, true);
+			Zotero.RDF.addStatement(resource, dc + "identifier", "ISBN " + item.ISBN, true);
 		}
 		if (item.ISSN) {
-			Zotero.RDF.addStatement(resource, dc+"identifier", "ISSN "+item.ISSN, true);
+			Zotero.RDF.addStatement(resource, dc + "identifier", "ISSN " + item.ISSN, true);
 		}
 		if (item.DOI) {
-			Zotero.RDF.addStatement(resource, dc+"identifier", "DOI "+item.DOI, true);
+			Zotero.RDF.addStatement(resource, dc + "identifier", "DOI " + item.DOI, true);
 		}
 		
 		// callNumber
 		if (item.callNumber) {
-			Zotero.RDF.addStatement(resource, dc+"identifier", item.callNumber, true);
+			Zotero.RDF.addStatement(resource, dc + "identifier", item.callNumber, true);
 		}
 		
 		// archiveLocation
 		if (item.archiveLocation) {
-			Zotero.RDF.addStatement(resource, dc+"coverage", item.archiveLocation, true);
+			Zotero.RDF.addStatement(resource, dc + "coverage", item.archiveLocation, true);
 		}
 		
 		// medium
 		if (item.medium) {
-			Zotero.RDF.addStatement(resource, dcterms+"medium", item.medium, true);
+			Zotero.RDF.addStatement(resource, dc + "medium", item.medium, true);
```

Comment:
There is no `medium` property in DC elements http://purl.org/dc/elements/1.1/ but in DC terms http://purl.org/dc/terms . However, this translator is named "Unqualified Dublin Core RDF" which would be just DC elements. Hm... we have to think this further.

Reply:
Actually, I think we can use `dc:format` here, which stands for "The file format, physical medium, or dimensions of the resource." http://www.dublincore.org/specifications/dublin-core/dcmi-terms/#elements-format . But this then should go into a separate commit.

Reply:
OK, so what should I do in the interim?

Reply:
I will handle that...